hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/recon/ReconConfigKeys.java

```diff
@@ -20,11 +20,12 @@
 
 /**
  * This class contains constants for Recon related configuration keys used in
- * SCM & Datanode.
+ * SCM &amp; Datanode.
```

Comment:
nit: "and" is simpler, and more readable in code view.

```suggestion
 * SCM and Datanode.
```

Reply:
Done